### DISCUSSION TITLE
Search and answer

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3533] [General]
I looked article [https://www.astronomy.com/science/the-planet-lhs-3154-b-seems-way-too-big-for-its-tiny-star/](url) **This newfound planet seems way too big for its tiny star**. I tried to find **LHS 3154** with Stellarium. F3 search found Star **LP 224-38**. There is no mention on the internet that they are the same item under different names. True. They are different.

So this is a misleading feature in Stellarium, if not an outright bug?

**Comment by gzotti** [comment]
Just go to the second tab in the F3 panel and retrieve all names from SIMBAD (not from Stellarium. We don't catalog such critters). You see both names, and a few more. Nothing to worry about.
